### ISSUE TITLE
Latest version not consistent.

### ISSUE THREAD

**Issue by barambani** [scalacenter/scaladex#548]
I notice that the link below to one of our application

https://index.scala-lang.org/laserdisc-io/laserdisc/laserdisc-fs2

doesn't select the latest available version for it while a similar one linked by

https://index.scala-lang.org/laserdisc-io/laserdisc/laserdisc-core

correctly selects the latest (that at the time of this message is 1.4). 
Do I have a workaround to force the page to always go to the latest available version ?
Thanks

**Comment by MasseGuillaume** [comment]
Yes you can login via GitHub an edit to project to only use semantic version

**Comment by barambani** [comment]
they both look on in the settings
<br>
<img width="519" alt="screen shot 2018-07-15 at 21 09 53" src="https://user-images.githubusercontent.com/201339/42737946-83a9fd8a-8873-11e8-961f-285559390c94.png">
